docs/python/debugging.md

```diff
@@ -163,11 +354,11 @@ When set to `true`, breaks the debugger at the first line of the program being d
 
 Specifies how program output is displayed.
 
-| Value | Where output is displayed |
-|--- | --- |
-| `"internalConsole"` | VS Code debug console |
+| Value                            | Where output is displayed                                          |
+|----------------------------------|--------------------------------------------------------------------|
+| `"internalConsole"`              | VS Code debug console                                              |
 | `"integratedTerminal"` (default) | [VS Code Integrated Terminal](/docs/editor/integrated-terminal.md) |
-| `"externalTerminal"` | Separate console window |
+| `"externalTerminal"`             | Separate console window                                            |
```

Comment:
This is not entirely correct. The Debug Console contains *redirected* output, which is controlled by "redirectOutput" debug property. This defaults to true if you use "internalConsole", and false otherwise, but it can be manually overridden by the user. If you use it together with either terminal option, then the output will be both in the terminal window, and in Debug Console. Conversely, if you disable it for internal console, then no output will be displayed at all.

Reply:
What @int19h said in table form:

| redirectOutput | console | Effect |
|----------------- | ----------------- | -------- |
| true (default for internalConsole) | internalConsole | Output is only displayed in the Debug Console  |
| false                 | internalConsole | Output is NOT displayed anywhere  |
| true                  | integratedTerminal or externalTerminal | Output is displayed both in Debug Console and Terminal |
| false (default for  integratedTerminal or externalTerminal) | integratedTerminal or externalTerminal | Output is only displayed in the Terminal |

Reply:
Thanks, I tried to clarify this in the info for console and redirectOutput. Look better?